index.js

```diff
@@ -1,12 +1,113 @@
 const express = require('express');
-const helmet = require('helmet');
+const configMiddleware = require('./config/middleware.js');
+
+const knex = require('knex');
+const knexConfig = require('./knexfile.js');
 
 const server = express();
 
-server.use(express.json());
-server.use(helmet());
+// apply middleware
+configMiddleware(server);
+
+// connect to database
+const db = knex(knexConfig.development);
+
+// generic 500 error handler
+const errorHandler = (err) => res.status(500).json({
+    message: 'Sorry, an error occurred while handling your request',
+    error: err
+  });
+
+// endpoints start here
+
+// list zoos
+server.get('/api/zoos', async (req, res) => {
+
+  try {
+    const zoos = await db('zoos');
+    res.status(200).json(zoos);
+  } catch (err) {
+    errorHandler(err);
+  }
+
+});
+
+// get zoo by ID
+server.get('/api/zoos/:id', async (req, res) => {
+
+  const { id } = req.params;
+
+  try {
+    const zoo = await db('zoos')
+      .select()
+      .from('zoos')
+      .where('id', id);
```

Comment:
Since the column name that you're looking for is the same as your variable name here, you could shorten it down a little by passing in an object:
```javascript
.where({ id });
```